vendor/models/offer.py

```diff
@@ -230,14 +235,54 @@ def get_trial_days(self):
 
     def has_any_discount_or_trial(self):
         if self.discount() or self.get_trial_amount() or\
-           self.get_trial_occurrences() or self.get_trial_days():
+           self.get_trial_occurrences() or self.get_trial_days() or\
+           (not self.billing_start_date and self.billing_start_date > timezone.now()):
```

Comment:
should this be `self.billing_start_date and self.billing_start_date > timezone.now()`? it seems the current logic will always evaluate to false; if billing_start_date is not set, it will never be greater than the current datetime

Reply:
If billing_start_date is not set that and block should always return False.

Reply:
thanks @mgcamisa you were right, I needed to remove that not thanks :)